content/en/agent/proxy.md

```diff
@@ -27,7 +27,105 @@ A few options are available to send traffic to Datadog over SSL/TLS for hosts th
 
 ## Web proxy
 
-Traditional web proxies are supported natively by the Agent. If you need to connect to the Internet through a proxy, edit your Agent configuration file.
+### Squid
+
+[Squid][2] is a forward proxy for the web supporting HTTP, HTTPS, FTP, and more. It runs on most available operating systems, including Windows, and is licensed under the GNU GPL. Squid is easy to configure and widely used for proxying traffic in a variety of different situations. 
+
+This is a simple, straightforward option if you do not already have a running web proxy in your network.
+
+#### Proxy forwarding with Squid
+
+##### Squid configuration
+
+Squid should be installed on a host that has connectivity to Datadog. Use your operating system's package manager, or install the software directly from [Squid's project page][2].
+
+Most of the settings below are configured within Squid's configuration file, usually located at `/etc/squid/squid.conf` on Linux or `C:\squid\etc\squid.conf` in Windows. 
+
+###### Configure Squid to only send traffic to Datadog
+
+Edit your `squid.conf` file so Squid is able to accept local traffic and forward it to the necessary Datadog intakes.
+
+An example config file is shared below, some of these ACLs (Access Lists) may already be defined:
+
+```conf
+http_port 0.0.0.0:3128
+
+acl local src 127.0.0.1/32
+
+acl Datadog dstdomain .{{< region-param key="dd_site" >}}
+
+http_access allow Datadog
+http_access allow local manager
+```
+
+###### Start Squid
+
+Restart Squid so that your configurations can be applied. If you're configuring Squid on Windows, you will first need [to configure Squid as a system service][9].
```

Comment:
```suggestion
Start (or restart if already started) Squid so that your new configurations can be applied. Note that if you are configuring Squid on Windows, you will first need [to configure Squid as a system service][9].
```